README.md

```diff
@@ -15,3 +15,40 @@ To run the site locally with Docker, use the following command:
 ```bash
 docker run --rm --label=jekyll --volume=$(pwd):/srv/jekyll  -it -p 127.0.0.1:4000:4000 jekyll/jekyll jekyll serve
 ```
+
+
+
+#### Side note for those on non-linux machines
+ 
```

Comment:
question: do I need minikube to view the site locally? won't docker-machine work?
Specifically, how will you share files from host to minikube? Don't you need to setup shared folders etc. in virtual box?